examples/chat/module/rust/src/lib.rs

```diff
@@ -59,14 +59,14 @@ impl OakNode for Node {
 
 fn unknown_id_err(which: &str) -> grpc::Result<()> {
     let mut status = oak::proto::status::Status::new();
-    status.set_code(3); // INVALID_ARGUMENT
+    status.set_code(grpc::Code::INVALID_ARGUMENT.value());
```

Comment:
We should probably have a helper method that takes a Code and a message and creates a Status, worth creating a tracking issue?

Reply:
Good idea, I've added it into this PR - PTAL.